src/mapml/control/ScaleBar.js

```diff
@@ -0,0 +1,48 @@
+export var ScaleBar = L.Control.extend({
+    options: {
+      metric: true,
+      imperial: true,
+      maxWidth: 100,
+      updateWhenIdle: true,
+      position: 'bottomleft'
+    },
+
+    onAdd: function (map) {
+      let container = L.DomUtil.create('div','leaflet-control-scale');
+
+      let scaleControl = L.control.scale(this.options);
+      scaleControl.addTo(map);
+      container.appendChild(scaleControl.getContainer());
+      container.title = "Scale bar";
```

Comment:
I think this announces "Scale bar" twice, because below you add `aria-label` and put "Scale bar: blah blah blah" in there.  Maybe ditch the title unless it gets added automatically?